build-openmpi.sh

```diff
@@ -12,6 +12,7 @@ echo "NIX_OMPI_IMAGE is $NIX_OMPI_IMAGE"
 docker build \
        --build-arg ADDUSER="$ADDUSER" \
        --build-arg BASEOS="$BASEOS" \
+       --build-arg SSH_PRIVATE_KEY="$(cat ~/.ssh/id_rsa)" \
```

Comment:
Just in case some users don't want to use this ssh key with bitbucket (for whatever reason), I'd usggest doing something like: `--build-arg SSH_PRIVATE_KEY="$(cat $BITBUCKET_SSH_KEY)"`

You could still supply a default value for `$BITBUCKET_SSH_KEY` to be `~/.ssh/id_rsa`; see e.g. https://stackoverflow.com/questions/4437573/bash-assign-default-value

Note: I'm terrible at bash and haven't checked that what I typed above is valid, but should illustrate the idea.